DeepCat.js

```diff
@@ -152,12 +156,18 @@
 	function computeResponses( responses, newSearchTerms ) {
 		var i,
 			userParameters,
-			newSearchTermString;
+			newSearchTermString,
+			errorMessages = [];
 
 		for ( i = 0; i < responses.length; i++ ) {
 			userParameters = JSON.parse( responses[i]['userparam'] );
 			newSearchTermString = '';
 
+			if ( !responses[i]['result'] || responses[i]['result'].length == 0) {
+				errorMessages[0] = createErrorMessage( 'deepcat-error-unexpected-response', null );
```

Comment:
Because that would cause the same error message if the result was empty multiple times. This way, we get either an array with 0 or 1 elements regardless of the number of empty results.